docs/src/main/asciidoc/firestore.adoc

```diff
@@ -146,6 +146,13 @@ Cloud Firestore only supports filter components joined by AND, and the following
 * `less than`
 * `is null`
 
+Operations expecting List arguments:
+
+* `contains` (allows for combining up to 10 array-contains clauses on the same field with a logical OR)
```

Comment:
I'm confused by this "10" limit.
Can you just explain that limitations in the note below?

Reply:
Can you please address?